ssl/ssl_local.h

```diff
@@ -3019,6 +3019,10 @@ void ossl_ssl_set_custom_record_layer(SSL_CONNECTION *s,
      SSL_OP_SINGLE_ECDH_USE                   | \
      SSL_OP_EPHEMERAL_RSA                     )
 
+/* This option is undefined in public headers with no-dtls1-method. */
+#ifndef SSL_OP_CISCO_ANYCONNECT
+# define SSL_OP_CISCO_ANYCONNECT 0
+#endif
```

Comment:
This looks like the wrong fix to me. I suggest you remove the use of SSL_OP_CISCO_ANYCONNECT for a no-dtls1-method build.

Reply:
That would require duplicating the whole macro that uses it. I do not think it would be a better fix. Alternatively I could create a third macro - something like `MAYBE_SSL_OP_CISCO_ANYCONNECT` and use that one later.

```
#ifndef SSL_OP_CISCO_ANYCONNECT
# define MAYBE_SSL_OP_CISCO_ANYCONNECT 0
#else
# define MAYBE_SSL_OP_CISCO_ANYCONNECT SSL_OP_CISCO_ANYCONNECT
#endif
```

Is that really better though?

Reply:
Please note that use of `SSL_OP_CISCO_ANYCONNECT` is of course guarded inside the  `OPENSSL_NO_DTLS1_METHOD` ifndef in the code so I do not see any problem with the existing approach to fixing this.

Reply:
Agree with @t8m that this is the cleanest approach to fixing this - none of them are ideal